swan-lake/integration-tools/openapi-tool.md

```diff
@@ -10,18 +10,74 @@ intro: OpenAPI Specification is a specification that creates a RESTFUL contract
 
 Ballerina OpenAPI tool makes it easy for you to start the development of a service documented in an OpenAPI contract in Ballerina by generating a Ballerina service and client skeletons. It enables you to take the code-first API design approach by generating an OpenAPI contract for the given service implementation.
 
-The Ballerina OpenAPI tool support provides the following capabilities.
+## Usage
 
-> **Prerequisite:** Install the latest [Ballerina Swan Lake distribution](https://ballerina.io/downloads/).
+The Ballerina OpenAPI tool support provides the following capabilities.
 
  1. Generate Ballerina service/client stubs from a given OpenAPI contract file using the CLI tool.
  2. Export the OpenAPI definition from a given Ballerina service implementation using the CLI tool.
  3. Validate the service implementation compliance with a provided OpenAPI contract using the OpenAPI annotation.
-- The OpenAPI compiler plugin allows you to validate a service implementation against an OpenAPI contract during the compile time. This plugin ensures that the implementation of a service does not deviate from its OpenAPI contract. 
+  >**Info:** The OpenAPI compiler plugin allows you to validate a service implementation against an OpenAPI contract during the compile time. This plugin ensures that the implementation of a service does not deviate from its OpenAPI contract. 
+
+### OpenAPI to Ballerina usage
+
+The OpenAPI to Ballerina command supports several usages in the Ballerina OpenAPI tool as follows.
+
+```
+$ bal openapi [-i | --input] <openapi-contract-file-path> 
+            [-o | --output] <output-location>
+            [--mode] <mode-type>
+            [--tags] <tag-names> 
+            [--operations] <operation-names> 
+            [-n | --nullable]
+            [--license] <license-file-path> 
+            [--with-tests]
+```
+
+### Ballerina to OpenAPI usage
+
+The Ballerina to OpenAPI command supports several usages in the Ballerina OpenAPI tool as follows.
+
+```
+$ bal openapi [-i | --input] <ballerina-service-file-path> [--json]
+            [-s | --service] <current-service-name>
+            [-o | --output] <output-location>
+```
+
+## Available parameters
```

Comment:
```suggestion
## Command Options
```

Reply:
@nirmal070125 @isuruh15 - This originated from the Health tool page. Can you check and confirm?

Reply:
+1